tsdb_k8s_queries/challenges/append-no-conflicts-rampup.json

```diff
@@ -0,0 +1,187 @@
+{
+  "name": "append-no-conflicts-rampup",
+  "description": "Indexes the whole document corpus. Ramps up the number of clients with each timed operation.",
+  "default": false,
+  "schedule": [
+    {
+      "name": "delete-all-datastreams",
+      "operation": {
+        "operation-type": "delete-data-stream"
+      }
+    },
+    {
+      "name": "delete-all-composable-templates",
+      "operation": {
+        "operation-type": "delete-composable-template"
+      }
+    },
+    {
+      "name": "create-all-templates",
+      "operation": {
+        "operation-type": "create-composable-template",
+        "request-params": {
+          "create": "true"
+        }
+      }
+    },
+    {
+      "name": "put-timestamp-pipeline",
+      "operation": {
+        "operation-type": "put-pipeline",
+        "id": "timestamp_pipeline",
+        "body": {
+          "processors": [
+              {
+                  "set": {
+                      "field": "@timestamp",
+                      "value": {{'"{{_ingest.timestamp}}"'}}
+                  }
+              },
+              {
+                "script": {
+                  "source": "Random r  = new Random();ctx['kubernetes']['pod']['uid'] = 'value-' + r.nextLong();"
+                }
+              }
+          ]
+        }
+      }
+    },
+    {
+      "name": "check-cluster-health",
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "{{cluster_health | default('green')}}",
+          "wait_for_no_relocating_shards": "true"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "parallel": {
+        "tasks": [
+          {%- set p_bulk_indexing_clients = bulk_indexing_clients | default(8) %}
+          {%- set split_bulk_indexing_clients = p_bulk_indexing_clients / 2 %}
+          {%- set split_bulk_indexing_clients = split_bulk_indexing_clients | int %}
+          {
+            "operation": "index-container",
+            "clients": {{split_bulk_indexing_clients}}
+          },
+          {
+            "operation": "index-pod",
+            "clients": {{split_bulk_indexing_clients}}
+          }
+        ]
+      }
+    }
+    {# serverless-post-ingest-sleep-marker-start #}{%- if post_ingest_sleep|default(false) -%}
+    {
+      "name": "post-ingest-sleep",
+      "operation": {
+        "operation-type": "sleep",
+        "duration": {{ post_ingest_sleep_duration|default(30) }}
+      }
+    }
+    {%- endif -%}{# serverless-post-ingest-sleep-marker-end #}
```

Comment:
Cosmetic remark: So far, the `post-ingest-sleep` operation was introduced in all the places where force-merge was present (as a simple replacement), in benchmarks where there's a clear-cut division between ingest and search. As this challenge is all about ramp-up and there's no clear moment when ingest stops I would skip this option.